pkgs/games/cgoban/default.nix

```diff
@@ -1,15 +1,15 @@
 { lib
 , stdenv
 , writers
-, adoptopenjdk-jre-bin
+, temurin-jre-bin-17
```

Comment:
Would be much better to have this receive a `jre` parameter and set the desired version in `all-packages.nix`, but I think this is not the focus of this PR.

Reply:
> Would be much better to have this receive a `jre` parameter and set the desired version in `all-packages.nix`, but I think this is not the focus of this PR.

There was no `jre` argument before this PR, so I think this shouldn't be a blocker - if someone really wants to replace the jre used by this package they anyway are recommended to read the source code.

Reply:
Definitely not a blocker, I was just talking that it would be nicer.